vault/auth.go

```diff
@@ -800,12 +800,9 @@ func (c *Core) setupCredentials(ctx context.Context) error {
 		backend, entry.RunningSha256, err = c.newCredentialBackend(ctx, entry, sysView, view)
 		if err != nil {
 			c.logger.Error("failed to create credential entry", "path", entry.Path, "error", err)
-			plug, plugerr := c.pluginCatalog.Get(ctx, entry.Type, consts.PluginTypeCredential, "")
-			if plugerr == nil && plug != nil && !plug.Builtin {
-				// If we encounter an error instantiating the backend due to an error,
-				// skip backend initialization but register the entry to the mount table
-				// to preserve storage and path.
-				c.logger.Warn("skipping plugin-based credential entry", "path", entry.Path)
+
+			if c.isMountable(ctx, entry, consts.PluginTypeCredential) {
+				c.logger.Warn("skipping plugin-based mount entry", "path", entry.Path)
```

Comment:
nit: I don't think it really matters very much, but I assume this change was not intentional.

```suggestion
				c.logger.Warn("skipping plugin-based credential entry", "path", entry.Path)
```